Better optional argument handling, including an optional `SafariTabs.db` path.

yig opened the pull request.

yig committed:
Ruby 3.4.7. arguments: path/to/SafariTabs.db path/to/output

yig committed:
better argument handling

yig committed:
proper argument handling with three optional arguments. Kept `-stp` instead of `--stp` for compatibility.

mokolabs commented:
Hey there! Thanks for submitting a PR!

Can you summarize these changes for me? I think I understand them, but want to make sure before merging anything.

Thank you!

yig commented:
Sorry for the description-less pull request. I added optional flags to customize the export location (`--out file/path`) and path to the `SafariTabs.db` (`--db path/to/SafariTabs.db`). I added the following to the README:

- Need to customize the export location? Pass `--out file/path` to the tabby command, as in:
  `ruby tabby.rb --out ~/Library/Backup`
- Need to run on a specific `SafariTabs.db`? Pass `--db path/to/SafariTabs.db` to the tabby command, as in:
  `ruby tabby.rb --db ~/Backups/SafariTabs.db`

mokolabs reviewed (approved):
Nice job. Thank you!
